resource/bikBitvArrayMap.json

```diff
@@ -0,0 +1,2681 @@
+{

```

Comment:
Where this file comes from?

Reply:
i created it manually. I investigated the diffrence beetwen the BITV Standard and the WCAG techniques. The result was the the BITV test steps and the WCAG techniques was similar if not even the same.